Pickers/AbstractActionSheetPicker.m

```diff
@@ -218,7 +254,7 @@ - (void)showActionSheetPicker {
 
 #pragma clang diagnostic push
 #pragma ide diagnostic ignored "UnavailableInDeploymentTarget"
-    if ([UIViewController instancesRespondToSelector:@selector(edgesForExtendedLayout)]) {
```

Comment:
No it has to be removed for better user experience on iOS < 8.0 (on which edgesForExtendedLayout is not available). Inclusion of this line (as it is now) disables tapDismiss action completely on older iOS. 

Inclusion of this line will stop execution of switch statement on iOS < 8.0 thus disabling tapDismiss action completely.